packages/protocol/scripts/truffle/invite.ts

```diff
@@ -83,4 +101,10 @@ module.exports = async (callback: (error?: any) => number) => {
   } catch (error) {
     callback(error)
   }
+  logTime('Finish')
+}
+
+function logTime(msg: string) {
+  const millis = Date.now() - start
+  console.log(`${millis} ms: ${msg}`)
```

Comment:
Is timing information useful for folks running the invite script?

Reply:
I am not sure. I decided to leave it in there since it might turn out to be useful. If it isn't removal is pretty straightforward.